src/Entity/Personne/Membre.php

```diff
@@ -54,6 +55,15 @@ class Membre implements AnonymizableInterface
      */
     private $personne;
 
+    /**
+     * @Assert\Valid()
+     *
+     * @ORM\OneToOne(targetEntity="App\Entity\Hr\Alumnus", mappedBy="personne",
+     *                                                                    cascade={"persist", "remove"},
+     *                                                                   orphanRemoval=true)
+     */
+    private $alumnus;
```

Comment:
Ce champ doit être annoté `@Groups({"gdpr"})` pour que l'export soit également appliqué sur `alumni`

Reply:
See 74eafd543bc9efe4e33290c7837c62a2c55844ed
Also change `access.html.twig` template
The `if` condition is a bit cumbersome but still maintainable with only 3 attributes.
Is it normal that on the `access.html.twig` template I can access non-gdpr attributes from `alumnus` class ?